src/components/blocks/metadataBlock/MetadataBlock.tsx

```diff
@@ -1,14 +1,17 @@
+import { ReactNode } from "react";
+
 import { Section } from "@/components/molecules/section/Section";
 import { IMetadata } from "@/types";
 
 interface IProps {
-  title: string;
+  title?: ReactNode;
   metadata: IMetadata[];
+  id?: string;
 }
 
-export const MetadataBlock = ({ title, metadata }: IProps) => {
+export const MetadataBlock = ({ title, metadata, id = "section-metadata" }: IProps) => {
   return (
-    <Section title={title} id="section-metadata">
+    <Section title={title} id={id}>
```

Comment:
question: what's this ID used for? I can't see it's use anywhere?

suggestion: if we're using it defaulting on parameters can be dangerous in the sense of if two clients don't pass an ID you get the same one (no longer an ID).

Reply:
Yes fair point RE: duplicate ids! The use case for these is they are the anchor points for the sidebar navigation which is used for navigating to a specific block on the page.